packages/design-system/src/components/DateField/SingleInputDateField.stories.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import SingleInputDateField from './SingleInputDateField';
+
+export default {
+  title: 'Components/SingleInputDateField',
+  component: SingleInputDateField,
+  argTypes: {
+    errorMessage: {
+      control: { type: 'text' },
+    },
+    hint: {
+      control: { type: 'text' },
+    },
+    label: {
+      control: { type: 'text' },
+    },
+    requirementLabel: {
+      control: { type: 'text' },
+    },
+  },
+  args: {
+    label: 'Birthday',
+    hint: 'Please enter your birthday',
+    name: 'single-input-date-field',
+  },
+};
+
+const Template = ({ ...args }) => {
+  const [dateString, setDateString] = useState('');
+  return <SingleInputDateField {...args} value={dateString} onChange={setDateString} />;
+};
+
+export const Default = Template.bind({});
+
+export const WithPicker = Template.bind({});
```

Comment:
Should this story contain some kind of `onChange` to better simulate what happens when a date is selected from the picker?

Reply:
Nevermind! I think my branch was behind because I see something happening when a date is selected in the picker